docker/celeryworker/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM nvidia/cuda:9.2-base-ubuntu18.04
 
-RUN apt-get update
-RUN apt-get install -y python3.6 python3-pip python3-dev build-essential libssl-dev libffi-dev libxml2-dev libxslt1-dev zlib1g-dev
-RUN apt-get install -y git curl netcat
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends python3-dev python3-pip python3-setuptools gcc git && \
```

Comment:
As you are not installing recommended package anymore `python3-wheel` is not installed anymore breaking the docker build (at least locally).

Removing `build-essentials` also seems to break the build as you need `gcc` and `g++` to build the python packages as they are not pre-built for the cuda image as they are for an ubuntu image for example.

Reply:
@AlexandrePicosson Good catch ! I must have made a mistake when I tested.